### PR TITLE
Dex 863 z indexing of alerts

### PR DESCRIPTION
**Pull request by Atticus29**
The problem was that for annotation creation, the dialog is full screen, so the alert was popping up in a part of the dialog that it shouldn't have been.

### CODE REVIEW

**Review by Emily-Ke — APPROVED** [review]
It's not quite consistent with our other dialogs.

**Comment by Atticus29** [comment]
> 

It's the only <StandardDialog> that has a fullscreen prop that also has an alert, so I kinda just decided what looked best 🤷‍♂️ .

File: src/components/AnnotationCreator.jsx [hunk 1]
```diff
@@ -249,6 +230,15 @@ export default function AnnotationCreator({
...
       </DialogActions>
+      {error && (
+        <CustomAlert
+          titleId="SERVER_ERROR"
+          style={{ marginTop: 16, marginBottom: 8 }}
+          severity="error"
+        >
+          {error}
+        </CustomAlert>
+      )}
```
Comment (brmscheiner) [inline src/components/AnnotationCreator.jsx:241]: does it look weird for the error to be below the buttons? everywhere else we do the errors above them

Comment (Atticus29) [reply src/components/AnnotationCreator.jsx]: Ok... moved it above. It looks better, I agree:
Screen Shot 2022-04-07 at 5 29 34 PM

File: src/components/AnnotationCreator.jsx [hunk 2]
```diff
@@ -207,6 +188,23 @@ export default function AnnotationCreator({
...
         </FormControl>
       </div>
+      {error && (
+        <CustomAlert
+          titleId="SERVER_ERROR"
+          style={{
+            marginTop: 16,
+            marginBottom: 8,
+            marginLeft: 'auto',
+            marginRight: 'auto',
```
Comment (brmscheiner) [inline src/components/AnnotationCreator.jsx:198]: you can abbreviate this `margin: '16px auto 8px auto'`

**Review by brmscheiner — APPROVED** [review]

**Pull request merged**